docs/serving/load-balancing/activator-capacity.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Configuring Activator capacity"
+linkTitle: "Configuring Activator capacity"
+weight: 50
+type: "docs"
+---
+
+# Configuring Activator capacity
+
+If there's more than one Activator in the system, Knative will put as many of them on the request path as it thinks is necessary to handle the current request load plus the target burst capacity. If the target burst capacity is 0, Knative only puts the Activator into the request path if the Revision is scaled to zero.
+
+If available, Knative will pick at least two Activators for high availability reasons. The actual number of Activators is calculated via a given _Activator capacity_ using the formula: `(replicas * target + targetBurstCapacity)/activatorCapacity`. That means, there are enough Activators in the routing path to handle the theoretical capacity of the existing application, including any additional target burst capacity.
```

Comment:
```suggestion
Knative uses at least two Activators to enable high availability if possible. The actual number of Activators is calculated as the _Activator capacity_, by using the formula `(replicas * target + targetBurstCapacity)/activatorCapacity`. This means that there are enough Activators in the routing path to handle the theoretical capacity of the existing application, including any additional target burst capacity.
```

Reply:
This is not quite accurate. The actual number is not the Activator capacity, but the Activator capacity is a factor in the formula.

Reply:
Updated it slightly

Reply:
I've already changed the wording a bit, please check if that's fine too.